.editorconfig

```diff
@@ -0,0 +1,133 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+# C# files
+[*.cs]
+# New line preferences
+csharp_new_line_before_open_brace = all
+csharp_new_line_before_else = true
+csharp_new_line_before_catch = true
+csharp_new_line_before_finally = true
+csharp_new_line_before_members_in_object_initializers = true
+csharp_new_line_before_members_in_anonymous_types = true
+csharp_new_line_between_query_expression_clauses = true
+
+# Indentation preferences
+csharp_indent_block_contents = true
+csharp_indent_braces = false
+csharp_indent_case_contents = true
+csharp_indent_switch_labels = true
+csharp_indent_labels = one_less_than_current
+
+# avoid this. unless absolutely necessary
```

Comment:
I'd love to open this (no pun intended) for wider discussion.   Personally, I'm a fan of explicit scope using `this.XX` and `TypeName.XX`

Reply:
Given the other rules about using `_` for members there isn't much of a need for having `this.` because it is obvious from the name things are members. If we decide against using `_`, which I see you also suggest, then I agree we should consider using `this.`.

Reply:
... That pretty much sums up my preferences.    Steer clear of psudo-Hungarian prefixes in favor of the constructs in the language to clarify the scope.